content/en/blog/2022/metrics-announcement.md

```diff
@@ -0,0 +1,37 @@
+    - Developers of applications within a technology organization, who want to capture metrics generated by OpenTelemetry APIs within their applications, either by their own developers or from shared libraries that their applications depend on. These metrics can be exported through OTLP or exported through any other OpenTelemetry exporter.
+    
+- Collector support for metrics, which includes the Collector’s ability to capture metrics from receivers for sources like host metrics or pre-packaged applications (these were already present in the Collector), the ability to receive metrics from protocols like OTLP and Prometheus, configuration-driven metric processing, and export these metrics to any supported destination. This is useful for:
+    - Anyone who wants to capture metrics from their hosts (Linux VMs, Windows, VMs, Kubernetes, etc.) or pre-packaged applications (databases, message queues, etc.).
+    - Anyone who wants to capture metrics from existing sources like OTLP (from OpenTelemetry SDKs, pre-packaged applications, etc.), Prometheus, or others.
+    - Anyone who wants to process / modify metrics and metric metadata captured from these sources.
+    - Anyone who wants to convert metrics from one format to another. For example, the Collector can capture metrics from a mix of OTLP and Prometheus sources, and then send all of these to a single destination using OTLP (with the standard OpenTelemetry semantic conventions), Prometheus, or any other exporter.
+    
+- Full OpenTelemetry Protocol (OTLP) support for efficiently serializing and transmitting metrics between systems.
+- A metrics section of the specification, which defines different types of metrics, their shapes, how to process them, and semantic conventions. This is primarily used by OpenTelemetry contributors but also provides guidance to OpenTelemetry users who are authoring metrics or metadata.
+
+All of this functionality is additive to OpenTelemetry’s existing tracing support, and both signal types share the same metadata and semantic conventions. As of this announcement, the Java, .Net, and Python SDKs have issued release candidates for OpenTelemetry’s metric capabilities, JS is coming next week, and more languages will be issuing metrics release candidates throughout the coming months. Each of these releases will be followed by general availability after we receive feedback from users.
+
+## Getting Started
+If you’re already using a mix of the OpenTelemetry APIs, SDKs, language agents, or Collector, then you can access GA metrics functionality by updating your OpenTelemetry artifacts to their latest versions. We're currently producing documentation and getting started guides for each artifact’s metrics capabilities within [the official OpenTelemetry documentation](../../../docs). Examples and supplementary documentation are also being added to each artifact’s corresponding GitHub repository.
```

Comment:
(Another reason why line-wrapping is good: it then becomes easier to pin specific review comments. In the meantime, I'll quote the text in the following series of comments on this paragraph.)

> or Collector

-->

> and Collector

Reply:
> you can access GA metrics

GA or RC?

Reply:
Replace `../../../docs` with just `/docs/`

Reply:
Consider changing:

> We're currently producing documentation ...

to

> We're currently updating [the official OpenTelemetry documentation](/docs/) for each artifact’s metrics capabilities.

Reply:
```suggestion
If you’re already using a mix of the OpenTelemetry APIs, SDKs, language agents, and Collector, then you can access the release candidate metrics functionality by updating your OpenTelemetry artifacts to their latest versions. We're currently updating [the official OpenTelemetry documentation](/docs) for each artifact's metrics capabilities. Examples and supplementary documentation are also being added to each artifact’s corresponding GitHub repository.
```